### DISCUSSION TITLE
Implement folding as built in feature or as an online tool.

### DISCUSSION THREAD

**Discussion by Klubbheads777** [PrismJS/prism#3544] [Ideas]
Hi everybody!
My idea is folding as a built in feature or as an online tool. I already developed such thing for my project at a job. If interested I may share code. [Attached zip removed by mod]

**Comment by mAAdhaTTah** [comment]
I removed the zip file because I don't really trust it. If you have code you want to share, please create a repo with it.

**Comment by Klubbheads777** [comment]
Sorry, I'm new to github

**Comment by Klubbheads777** [comment]
Here is permalink
https://github.com/Klubbheads777/prismSample/blob/fb13c46d2ecc0797f525418f6236a450963c5164/SyncParamEvents.html

**Comment by Klubbheads777** [comment]
![folding](https://user-images.githubusercontent.com/112972095/188903499-ad7a3eb5-70aa-448c-a3a5-5c619895b3b3.jpg)
